lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -846,17 +846,18 @@ int amf_app_handle_pdu_session_accept(
   memcpy(
       smf_msg->msg.pdu_session_estab_accept.qos_rules.qos_rule, &qos_rule,
       1 * sizeof(QOSRule));
-  convert_ambr(
-      &pdu_session_resp->session_ambr.downlink_unit_type,
-      &pdu_session_resp->session_ambr.downlink_units,
-      &smf_msg->msg.pdu_session_estab_accept.session_ambr.dl_unit,
-      &smf_msg->msg.pdu_session_estab_accept.session_ambr.dl_session_ambr);
 
-  convert_ambr(
-      &pdu_session_resp->session_ambr.uplink_unit_type,
-      &pdu_session_resp->session_ambr.uplink_units,
-      &smf_msg->msg.pdu_session_estab_accept.session_ambr.ul_unit,
-      &smf_msg->msg.pdu_session_estab_accept.session_ambr.ul_session_ambr);
+  // Set session ambr
+  smf_msg->msg.pdu_session_estab_accept.session_ambr.dl_unit =
+      smf_ctx->dl_ambr_unit;
+  smf_msg->msg.pdu_session_estab_accept.session_ambr.dl_session_ambr =
+      smf_ctx->dl_session_ambr;
+
+  smf_msg->msg.pdu_session_estab_accept.session_ambr.ul_unit =
+      smf_ctx->ul_ambr_unit;
+  smf_msg->msg.pdu_session_estab_accept.session_ambr.ul_session_ambr =
+      smf_ctx->ul_session_ambr;
+
```

Comment:
which unit test is covering these lined additions?

Reply:
These line are part of clean up and are covered by end to end procedure unit test in: lte/gateway/c/core/oai/test/amf/test_amf_procedures.cpp, TEST_F(AMFAppProcedureTest, TestPDUSessionSetup).